packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionFirstInvoice',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'subscriptionFirstInvoice',
+  'Sent to inform a user that their payment is currently being processed.',
+  {
+    productName: 'Firefox Fortress',
+    cardType: 'MasterCard',
+    icon: 'https://accounts-static.cdn.mozilla.net/product-icons/mozilla-vpn-email.png',
+    invoiceDateOnly: '10/13/2021',
+    invoiceLink:
+      'https://pay.stripe.com/invoice/acct_1GCAr3BVqmGyQTMa/invst_GyHjTyIXBg8jj5yjt7Z0T4CCG3hfGtp',
+    invoiceNumber: '8675309',
+    invoiceTotal: '$20.00',
+    lastFour: '5309',
+    nextInvoiceDateOnly: '11/13/2021',
+    payment_provider: 'stripe',
+    subscriptionSupportUrl: 'http://localhost:3030/support',
+  }
+);
+
+export const SubscriptionFirstInvoice = createStory();
```

Comment:
Maybe we can have stories that demos credit card, paypal, and neither(this possible?)

Reply:
+1 to Jody asking "this possible", I'm not sure that we need a "default" story without either, but maybe we do? Not really sure how else someone could pay besides Stripe/Paypal. (could poke around the mailer code and/or ask the subplat team)

Reply:
I have removed the "default" story as there was no conditional to include the `paymentProvider` partial in the older template. Stripe/PayPal are the only two options within that partial though.